helix-term/src/ui/editor.rs

```diff
@@ -205,7 +205,9 @@ impl EditorView {
             .find_scope_index("diagnostic")
             .or_else(|| theme.find_scope_index("ui.cursor"))
             .or_else(|| theme.find_scope_index("ui.selection"))
-            .expect("no selection scope found!");
+            .expect(
+                "could not find `diagnostic`, `ui.cursor`, and `ui.selection` scopes in the theme!",
```

Comment:
Nit: I would say "or" instead of "and", since you only need one of them in the config.

Reply:
The panic would only happen if you're missing all 3 scopes. Should it be changed to: 
> at least one of the following scopes must be defined in the theme: `diagnostic`, `ui.cursor`, or `ui.selection`!

Reply:
It should show 'or'

Reply:
> The panic would only happen if you're missing all 3 scopes.

Right, but using "and" in this case gives the impression that you need all of them, and that perhaps only one of them is missing.  Although, admittedly, English is ambiguous in this case, and this is really just a nit.  I don't mean to make a big deal out of it.

> at least one of the following scopes must be defined in the theme: diagnostic, ui.cursor, or ui.selection!

I really like that, actually!  That's really clear and concise.